backends/ebpf/runtime/ebpf_test.h

```diff
@@ -141,6 +139,17 @@ struct bpf_table tables[] = {
 #define BPF_OBJ_PIN(table, name) registry_add(table)
 #define BPF_OBJ_GET(name) registry_get_id(name)
 
+/*
+ * Helper function.
+ * Print a byte buffer according to the specified length.
+ */
+static inline void print_n_bytes(void *receiveBuffer, int num) {
```

Comment:
This function is meant to print bytes of any type of address. If I used const char it would likely complain about type mismatch. It is pretty useful for debug print statements, but I can also remove it.